bin/transform-gisaid

```diff
@@ -129,15 +172,32 @@ if __name__ == '__main__':
 
         if not args.sorted_fasta:
             pipeline = pipeline | DropSequenceData()
-
+        
         pipeline = (
             pipeline
             | ExpandLocation()
             | FixLabs()
             | AbbreviateAuthors()
             | ParsePatientAge()
             | ParseSex()
-            | AddHardcodedMetadata()
+            | AddHardcodedMetadata() 
+        )
+
+        # writing the raw metadata in a tsv file
+        pipeline = ( pipeline  | WriteCSV(RAW_METADATA_FILENAME, 
+                                            METADATA_COLUMNS , 
+                                            restval = '?' , 
+                                            extrasaction ='ignore' , 
+                                            delimiter  = '\t', 
+                                            dict_writer_kwargs  = {'lineterminator': args.newline} ) )
+
+
+        # applying the substitution rules (temporary : writing the intermediary data to verify effect )
+        dict_writer_kwargs = {'lineterminator': args.newline}
```

Comment:
Do we need to assign this here if it is reassigned before being used below https://github.com/nextstrain/ncov-ingest/pull/135/files#diff-5ffbfeffdca16cf94d3822dca1129f8644b0430390a76df2551eed4db94cfa33R232 ?